kafka_redis_fs/fs-kafka-redis.ipynb

```diff
@@ -1,21 +1,42 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    resp
</pre><p>last thing - at the end of the notebook can u please add purge_targets() to your feature set so there won't be any garbage left in our centralized redis db after the run. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/test-notebooks/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
Added purge_targets()